component/prometheus/exporter/postgres/postgres.go

```diff
@@ -0,0 +1,76 @@
+package postgres
+
+import (
+	"github.com/grafana/agent/component"
+	"github.com/grafana/agent/component/prometheus/exporter"
+	"github.com/grafana/agent/pkg/flow/rivertypes"
+	"github.com/grafana/agent/pkg/integrations"
+	"github.com/grafana/agent/pkg/integrations/postgres_exporter"
+	config_util "github.com/prometheus/common/config"
+)
+
+func init() {
+	component.Register(component.Registration{
+		Name:    "prometheus.exporter.postgres",
+		Args:    Arguments{},
+		Exports: exporter.Exports{},
+		Build:   exporter.New(createIntegration, "postgres"),
+	})
+}
+
+func createIntegration(opts component.Options, args component.Arguments) (integrations.Integration, error) {
```

Comment:
nit: We called these `createExporter` in the other exporters (process exporters would need to be changed too)

Reply:
Done, thanks!